src/sssom_schema/schema/sssom_schema.yaml

```
@@ -59,13 +63,15 @@ enums:
       rdf property:
         meaning: rdf:Property
       composed entity expression:
+        meaning: sssom:ComposedEntityExpression
         description: This value indicates that the entity ID does not represent a single entity, but a composite involving several individual entities. This value MUST NOT be used in the predicate_type slot. This specifications does not prescribe how an ID representing a composite entity should be interpreted; this is left at the discretion of applications.
         see_also:
         - https://github.com/mapping-commons/sssom/issues/402
         - https://github.com/mapping-commons/sssom/blob/master/examples/schema/composite-entities.sssom.tsv
         
   predicate_modifier_enum:
     permissible_values:
+      meaning: sssom:NegatedPredicate
       Not: Negating the mapping predicate. The meaning of the triple becomes subject_id is not a predicate_id match to object_id.
```

```suggestion
      Not: Negating the mapping predicate. The meaning of the triple becomes subject_id is not a predicate_id match to object_id.
        meaning: sssom:NegatedPredicate
```
Moved the *meaning* to be after the enum value and indented one more level.

---

:scream: Thanks for spotting that!